scipy/optimize/_differentialevolution.py

```diff
@@ -12,10 +12,11 @@
 from scipy.optimize._constraints import (Bounds, new_bounds_to_old,
                                          NonlinearConstraint, LinearConstraint)
 from scipy.sparse import issparse
-
+from scipy.stats import qmc
```

Comment:
This makes `optimize` depend on `stats`, which wasn't the case before. `stats` did already depend on `optimize` though. Apparently it doesn't create an import cycle, but it is slightly unfortunate. In other such situations, we've had to move the code that both submodules depend on to `scipy/_lib/`.

Something to look at a little more closely before merging.